src/FINDOLOGIC/Api/Responses/Xml21/Properties/Filter/Item/ImageItem.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace FINDOLOGIC\Api\Responses\Xml21\Properties\Filter\Item;
+
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
+use SimpleXMLElement;
+
+class ImageItem extends Item
```

Comment:
Please rename this to `VendorImageItem`, because the `vendor` is (apart from color item), the only one who can have an `image`.